Mod Menu Helper/assets/modmenu/lang/ko_kr.json

```diff
@@ -0,0 +1,55 @@
+{
+    "modmenu.dropInfo.line1": "FO 모드는 §9✎§r 또는 §8✎§r, 대체제는 §c*§r로 표시됩니다",
+    "modmenu.dropInfo.line2": "다른 모드는 지원되지 않습니다",
+    "modmenu.summaryTranslation.slight-gui-modifications": "애니메이션, 메뉴의 모드팩 버전\n§9✎§8 애니메이션, 색, FPS 인디케이터, ...",
+    "modmenu.summaryTranslation.advancementinfo": "커지고 스마트해진 도전과제 창\n§9✎§8 색, 정보 칸 크기, ...",
+    "modmenu.summaryTranslation.advancements-enlarger": "§c*§r 커진 도전과제 창\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.animatica": "Optifine의 애니메이션 텍스쳐\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.antighost": "/ghost 명령어로 Ghost 블록 제거\n§9✎§8 키바인드",
+    "modmenu.summaryTranslation.betterbeds": "§c*§r 빨라진 침대 렌더\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.bettermounthud": "말을 타고있을 때 허기와 경험치 표시\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.bobby": "서버에서 계속 청크가 보이도록 로드\n§9✎§8 블록 엔티티, 청크 어둡게, 캐시 저장, ...",
+    "modmenu.summaryTranslation.borderlessmining": "전체 화면에서 화면 이동 (opt-in)\n§9✎§8 모드 토글, macOS 지원, 커스텀 크기, ...",
+    "modmenu.summaryTranslation.capes": "Optifine과 기타 서드파티 망토\n§9✎§8 형태, 선호 형태, 겉날개, ...",
+    "modmenu.summaryTranslation.citresewn": "Optifine의 커스텀 아이템\n§9✎§8 모드 토글, 깨진 팩 허용, ...",
+    "modmenu.summaryTranslation.colormatic": "Optifine의 커스텀 색\n§9✎§8 맑은 하늘/공허, 부드러운 조명",
+    "modmenu.summaryTranslation.continuity": "Optifine의 연결된 텍스쳐\n§9✎§8 모드 토글, 메소드, 리소스 팩",
+    "modmenu.summaryTranslation.cullleaves": "Optifine의 \"스마트 나무\"\n§9✎§8 모드 토글, 리소스 팩",
+    "modmenu.summaryTranslation.cem": "Optifine의 커스텀 엔티티 (알파)\n§9✎§8 옵티파인 지원, 모델 고침",
+    "modmenu.summaryTranslation.dcch": "여러 월드에 걸쳐 입력된 기록 유지\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.dynamicfps": "백그라운드에서 FPS 감소\n§9✎§8 모드 토글, 호버시 복구, ...",
+    "modmenu.summaryTranslation.enhancedblockentities": "빠른 상자, 침대, 표지판 등 렌더\n§9✎§8 개선할 블록, 메소드",
+    "modmenu.summaryTranslation.entityculling": "벽 뒤 엔티티 렌더 안함\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.entity_texture_features": "Optifine의 랜덤/Emissive 몹\n§9✎§8 랜덤 몹, Emissivity, 눈 깜빡임, ...",
```

Comment:
"Emissive" as in "glowing" or "giving off light". Example: https://media.forgecdn.net/attachments/428/176/2022-01-25_23.png